client/src/components/AddCard/AddCard.tsx

```diff
@@ -0,0 +1,63 @@
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import { FormikHelpers } from 'formik';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import { setDefaultPayment } from '../../helpers/APICalls/setDefaultPayment';
+import AddCardForm from './AddCardForm/AddCardForm';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+export default function AddCard(): JSX.Element {
+  const classes = useStyles();
+  const { updateSnackBarMessage } = useSnackBar();
+  const { loggedInUser } = useAuth();
+
+  const handleSubmit = (
+    { paymentMethod }: { paymentMethod: string },
+    { setSubmitting }: FormikHelpers<{ paymentMethod: string }>,
+  ) => {
+    if (loggedInUser != undefined && loggedInUser) {
+      setDefaultPayment({ paymentMethod: paymentMethod, userId: loggedInUser.id }).then((data) => {
+        if (data.error) {
+          setSubmitting(false);
+          updateSnackBarMessage(data.error.message);
+        } else if (data.url) {
+          setSubmitting(false);
+          //window.open(data.url, '', 'width=1000,height=700');
```

Comment:
Let's get rid of this comment

Reply:
> Hey @samsoncsyu7777, please resort to using the proper Git Workflow. I already reviewed some changes here in another PR and it now makes code review take more time. Always select the right base branch for each PR. Also resolve the merge conflicts in this branch 🙏

In order to create the Stripe payment using its APIs, its documentation states that I should create a BE route, then create a FE component to input some sensitive information for the next route. That's why I can't prove my route work. And these 4 payment tickets are one based on one. Of course, I select the previous ticket as the base. However, the previous ticket hasn't been approved and I want to well use the time, so, I keep on the development on the next ticket. I think I may pull these changes from the previous ticket. How do you think?

Reply:
Thank you so much to provide me with the comments. I think it is a good idea to revise this branch with your comments instead of pulling from another branch at this moment.

Reply:
> > Hey @samsoncsyu7777, please resort to using the proper Git Workflow. I already reviewed some changes here in another PR and it now makes code review take more time. Always select the right base branch for each PR. Also resolve the merge conflicts in this branch 🙏
> 
> In order to create the Stripe payment using its APIs, its documentation states that I should create a BE route, then create a FE component to input some sensitive information for the next route. That's why I can't prove my route work. And these 4 payment tickets are one based on one. Of course, I select the previous ticket as the base. However, the previous ticket hasn't been approved and I want to well use the time, so, I keep on the development on the next ticket. I think I may pull these changes from the previous ticket. How do you think?

OK. No problems.